src/accordion/accordion.js

```diff
@@ -79,7 +79,9 @@ angular.module('ui.bootstrap.accordion', ['ui.bootstrap.collapse'])
     link: function(scope, element, attrs, accordionCtrl) {
       accordionCtrl.addGroup(scope);
 
+      scope.openClass = attrs.openClass || 'panel-open';
       scope.$watch('isOpen', function(value) {
+        element.toggleClass(scope.openClass, value);
```

Comment:
This is incorrect - if the element has a different class for some reason, this will cause a bug.

You will want
```javascript
element[value ? 'addClass' : 'removeClass'](scope.openClass, value)
```
in this case to be safe.